src/site/xdoc/checks.xml.template

```diff
@@ -27,2407 +27,8 @@
       </p>
       <div class="wrapper">
         <table>
-          <tr>
-            <td>
-              <a href="checks/naming/abbreviationaswordinname.html#AbbreviationAsWordInName">
-              AbbreviationAsWordInName</a>
-            </td>
-            <td>
               <macro name="checks">
```

Comment:
please name macro to be `allCheckSummarries` to be a bit more descriptive on what it doing.